### ISSUE TITLE
Unable to insert multiple variables into string value

### ISSUE THREAD

**Issue by mikeogezi** [electrikmilk/cherri#25]
Here is the baseline script:
```
@fname = "Cherri"
@sname = "MacBerry"
alert("First: {fname} Last: {sname}")
```

Here is what is produced:
<img width="617" alt="Screenshot 2023-10-31 at 6 22 05 AM" src="https://github.com/electrikmilk/cherri/assets/10839579/3eab9fb9-f879-4e36-ae4b-19c210633fd7">

As you can observe, the second variable is ignored. The same holds for more than two variables.

**Comment by electrikmilk** [comment]
Whoops! This might have been broken recently with a refactor to this functionality.

**Comment by electrikmilk** [comment]
I see how this didn't get caught, the variables test only uses one inline var ever. I'll fix that to mitigate something like this happening after a change again in the future.

Thanks for catching this!

**Comment by electrikmilk** [comment]
Fixed by [12a1efd](https://github.com/electrikmilk/cherri/commit/12a1efd18f50fb1d1793440c0fe4feeeda416a48)
